app/views/players/show.html.erb

```diff
@@ -94,4 +94,58 @@
       <% end %>
     </div>
   </div><br><br><br>
+<% end %>
+
+<% if @player.position == "Wide Receiver" %>
```

Comment:
Can you find anu opportunities for DRYing up some of the markup in this template?